### DISCUSSION TITLE
Create a group on meetup.com.

### DISCUSSION THREAD

**Discussion by HarshKapadia2** [bostonopen/bostonopen.github.io#3] [Ideas]
As I said in [another Discussion comment](https://github.com/orgs/bostonopen/discussions/2#discussioncomment-5526175) before:

> Creating a group on [meetup.com](https://meetup.com) would be nice, as a lot of people check that.
> 
> I usually find all my Tech events there and I know a lot of people who do that as well.

@pdurbin [replied](https://github.com/orgs/bostonopen/discussions/2#discussioncomment-5530816):

> ...(From a quick look, it seems to cost [$16.49 per month](https://help.meetup.com/hc/en-us/articles/360001620472-Organizer-subscription-pricing) and I'm not interested in paying that personally.)...

I didn't know that Meetup is always paid for groups. I thought they had a free plan.

**Comment by szabgab** [comment]
Older accounts on Meetup have a deal that they can have 3 groups in their account for the basic cost. You might be able to find a friendly organizer, maybe of another Boston-area tech group, who would be ready to use one of his/her empty Meetup slots for you.

**Reply by pdurbin** [reply]
Interesting. Not a bad idea, but I'd hate to be relying on some special deal, some grandfathered account.
